provisioning/openstack/CONFIGURE.md

```diff
@@ -16,54 +16,53 @@ Create or designate a suitable machine for running the PNDA CLI. We recommend Ce
 
 #### Obtain code
 
-Clone the [Heat templates repository](https://github.com/pndaproject/pnda-heat-templates) repository from the master branch at a specific release tag (e.g. ```release/3.5```) to the client machine.
+Clone the [pnda-cli repository](https://github.com/pndaproject/pnda-cli) repository from the master branch at a specific release tag (e.g. ```release/X.Y```, where X is major version > 4 and Y is a subversion) to the client machine.
```

Comment:
The additional text (where X is major, etc) here is misleading. There's no need to explain semantic versioning mid-sentence. The user simply needs to pick a valid release tag.